Source/Locks.csproj

```diff
@@ -25,7 +25,7 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
+    <OutputPath>..\Assemblies\</OutputPath>
```

Comment:
Puts the .dll in the right spot without having to copy it

Reply:
oh I don't use release anyway

Reply:
yea but in debug it works